app/views/works/_standard_form.html.erb

```diff
@@ -265,6 +265,7 @@
         <span class="html-notes"><%= allowed_html_instructions %></span>
         <span class="hidden rtf-notes"><%= ts('Type or paste formatted text.') %><%= link_to_help "rte-help" %></span>
       </p>
+      <p class="notice"><%= ts("<strong>Note:</strong> Text entered in the posting form is <strong>not</strong> automatically saved. Always keep a backup copy of your work.</p>")%>
```

Comment:
It looks like you're closing `</p>` tag is inside the translation string, not outside, and you'll need a space between the closing `)` and the `%>`. The translation will also need `.html_safe`, since it has `<strong>` tags in there.